addons/l10n_ar/models/account_move.py

```diff
@@ -267,6 +280,14 @@ def _l10n_ar_get_amounts(self, company_currency=False):
 
     def _get_vat(self):
```

Comment:
@zaoral check the missing argument that was there in V13 (company_currency=False)